apps/test/unit/code-studio/pd/foorm/FoormEditorTest.js

```diff
@@ -95,7 +104,7 @@ describe('FoormEditor', () => {
       JSON.stringify(sampleSaveData)
     ]);
 
-    const saveBar = wrapper.find('FoormSaveBar');
+    const saveBar = wrapper.find('FormSaveBar');
```

Comment:
i think these names need to be updated. also, in all the tests i've worked on that use `find`, we import anything we want to find so we can pass in the actual symbol for the component instead of a string (e.g. `FoormFormSaveBar` instead of `'FoormFormSaveBar'`). that seems much more robust imo, because if the component name changes you'd get a clear failed import error instead of what might be a more opaque error about not finding what you're expecting.

Reply:
Makes sense to me -- I had to make some test updates to make tests pass now that we're passing in FoormFormSaveBar as a prop and tried to implement this. For some reason, using find(FoormFormSaveBar) (component, not string) isn't working for me. Curious if you can see what I'm missing?

Reply:
ahhhh, TIL: when using `find` i've often found i need to add `.childAt(0)` to get what i want and i have never been quite sure why / under what circumstances. the default export from `FoormFormSaveBar` is the return value from the `connect` function, aka the "ConnectedFoormFormSaveBar", so that's what enzyme finds when it looks for that component. but the actual component is the first child of the connected component. 

to avoid confusion around when we do or do not need to use `.childAt(0)`, i think a better convention would be to always look for the unconnected version when we're testing the connected version, because then we never need to call `.childAt(0)`.

since i was testing this on your branch, i went ahead and added an `UnconnectedFoormFormSaveBar` and updated all the finds to use that instead of the display name. hope you don't mind!

Reply:
Great, thank you!